src/main/resources/META-INF/jpa-schema-export.sql

```diff
@@ -349,13 +347,6 @@
         primary key (id)
     );
 
-    create table Letter_Allophone (
-       Letter_id bigint not null,
-        allophones_id bigint not null,
-        allophones_ORDER integer not null,
-        primary key (Letter_id, allophones_ORDER)
-    );
-
```

Comment:
We'll also need to delete this table from the database. This can be done by adding a DB migration script. For an example see https://github.com/elimu-ai/webapp/blob/master/src/main/resources/db/migration/2001194.sql

Reply:
This too is done